StudentAdvisementSystem/web/studentFolder/profile.xhtml

```diff
@@ -6,18 +6,23 @@
 
 
     <h:head>
-            <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<!--            <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
             <script src="https://use.fontawesome.com/d9195f68cb.js"></script>
             <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css" />
-            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" type="text/css" /> 
-            <!--<link rel="stylesheet" href="assets/css/style.css"/>
-            <link rel="stylesheet" href="assets/css/form-elements.css"/>-->
-            <h:outputStylesheet name="css/style.css"/>
-            <h:outputStylesheet name="css/form-elements.css"/>
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" type="text/css" /> -->
+<!--            <h:outputStylesheet name="css/style.css"/>
+            <h:outputStylesheet name="css/form-elements.css"/>-->
             
             <title>Profile</title>
     </h:head> 
     <h:body>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```

Comment:
I was trying to fix the css and it says primefaces will overwrite unless you move the links in to the body. It seems that it is still overwriting the css